### ISSUE TITLE
Revise the help text in the "choose a password" box on publiclab.org/signup

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#6088]
### What part of publiclab.org is this a new feature idea for?

This feature idea applies to https://publiclab.org/signup while not signed in. 

### What is the idea? 

This idea is to change the help text in the "choose a password" box that someone sees when creating a new account on PublicLab.org. Using the help text to explain the password requirements would be an improvement on what we have now. 

### Details about this idea: 

The text inside the password field should indicate the password requirements that the system will accept. For instance, the current text simply says "enter your new password": 

<img width="711" alt="Screen Shot 2019-07-31 at 3 00 05 PM" src="https://user-images.githubusercontent.com/161439/62240034-1764c800-b3a4-11e9-8477-d367a19bbe86.png">

However, here's the message a person sees if they fail to type a password that meets system requirements; this message explains the password requirements that were never shown on the previous screen: 

<img width="717" alt="Screen Shot 2019-07-31 at 3 03 01 PM" src="https://user-images.githubusercontent.com/161439/62240180-5f83ea80-b3a4-11e9-8bc7-4d190730d2e7.png">

My feature request is that we use the text inside the password field to state the system requirements for the password, as shown in this example from Typeform: 

<img width="293" alt="Screen Shot 2019-07-31 at 3 04 22 PM" src="https://user-images.githubusercontent.com/161439/62240262-90641f80-b3a4-11e9-9ebc-650aa03476c2.png">

### What would need to be done:

- [ ] Find where we have stored the help text shown in the password box
- [ ] Delete the text "enter your new password"
- [ ] Add the text "password must be at least 8 characters"

**Comment by newbazz** [comment]
Or i think, there can be some dynamic checking done of the password while someone is typing. Unable the submit button until it is atleast 8 characters long.

**Comment by CleverFool77** [comment]
> Or i think, there can be some dynamic checking done of the password while someone is typing. Unable the submit button until it is atleast 8 characters long.

Hi @ebarry Wasn't this feature of password validation added earlier ? I did saw the PR I guess.

**Comment by ebarry** [comment]
cool idea @newbazz! 
@CleverFool77 -- thank you for letting me know! I had this experience last week, so if there's a PR already on the way -- amazing! Once it lands, i'll close this issue! Or -- please advise on the best path to proceed 🙏

**Comment by CleverFool77** [comment]
I did open one PR on this @ebarry which is this - https://github.com/publiclab/plots2/pull/5368 though it had many conflicts as This PR was for overall validation using `jquery-validation`  but when This was on review , at that time 3-4 four more PRs were opened, which were validation for particular field like mail or other stuff using either regex or different ways. And they are merged.
My aim to open this PR was to solve all the conflicts and form validation with just one `jqeyry-validation` but later as different ways of validation for different fields in different PRs got merged. 

I left this. So Would you like me to just take the password validation from this as validatoin for other fields is already there and solve this ?

**Comment by ebarry** [comment]
hi @CleverFool77 ! sorry just seeing this now. I have to admit i do not totally understand your comment, i apologize --- I am not sure what you mean about taking the password validation from this to validate other fields.  I do see that your PR #5368 is closed. 

Might we EITHER change the current help text in the password box itself (currently reads: "enter your new password") to the more helpful "password must be at least 8 characters" ? OR, perhaps, leave the box text unchanged, and add "password must be at least 8 characters" after the heading "create a password"?

**Comment by CleverFool77** [comment]
Hi liz, I understood this. I'll craate a PR soon. My earlier PR was closed
because it was covering various positions and password portion was one of
them. As other portions except password were merged by other contributors.
So I closed that pr.
So, Now I'll create a new PR which consists of password portion only. And
I'll put the test and validation accordingly.

**Comment by ebarry** [comment]
I really appreciate it @CleverFool77 !!!!!
